dip-template/runtimes/dip-provider/src/dip.rs

```diff
@@ -48,8 +48,7 @@ pub mod runtime_api {
 
 	#[derive(Encode, Decode, TypeInfo)]
 	pub enum DipProofError {
-		IdentityNotFound,
-		IdentityProviderError(<LinkedDidInfoProviderOf<Runtime> as IdentityProvider<DidIdentifier>>::Error),
+		IdentityProviderError(LinkedDidInfoProviderError),
```

Comment:
These errors are not following our convention not to have the term 'Error' in the suffix